python/cudf/cudf/core/dataframe.py

```diff
@@ -5713,6 +5713,43 @@ def isin(self, values):
         DataFrame:
             DataFrame of booleans showing whether each element in
             the DataFrame is contained in values.
+
+        Examples
+        --------
+        >>> import cudf
```

Comment:
Do we normally have import statements in examples?

Reply:
Yes, we usually have them because we have a `copy` button that lets a user copy the sample code snippet, and instead of the user intervening again for any potential import error we just add them in our samples.